packages/components/src/mobile/audio-player/styles.scss

```diff
@@ -0,0 +1,114 @@
+.container {
+	background-color: $light-ultra-dim;
+	border-radius: 4px;
+	flex-direction: row;
+	align-items: center;
+}
+
+.containerDark {
+	background-color: $dark-ultra-dim;
+}
+
+@mixin circle($width) {
+	width: $width;
+	height: $width;
+	border-radius: $width/2;
+}
+
+.iconContainer {
+	background-color: $light-ultra-dim;
+	margin: 12px 16px;
+	align-items: center;
+	justify-content: center;
+	@include circle(37px);
+}
+
+.iconContainerDark {
+	background-color: $dark-ultra-dim;
+}
+
+.icon {
+	color: $light-secondary;
+}
+
+.iconDark {
+	color: $dark-secondary;
+}
+
+.iconDisabled {
+	color: $light-quaternary;
+}
+
+.iconDisabledDark {
+	color: $dark-quaternary;
+}
+
+.disabledIcon {
+	color: $black;
+}
+
+.titleContainer {
+	padding: 12px 0;
+	flex: 1;
+}
+
+.titleContainerIOS {
+	margin-top: -1px;
+}
+
+.titleContainerAndroid {
+	margin-top: -2px;
+}
+
+.title {
+	font-size: 17;
+	margin-bottom: 1px;
+	color: $light-primary;
+}
+
+.titleDark {
+	color: $dark-primary;
+}
+
+.subtitleContainer {
+	flex-direction: row;
+	align-items: center;
+}
+
+.subtitle {
+	font-size: 12;
+	color: $light-secondary;
+}
+
+.subtitleDark {
+	color: $dark-secondary;
+}
+
+.buttonBackground {
+	margin: 12px 16px;
+	padding: 8px 16px;
+	border-radius: 500px;
+	background-color: $light-ultra-dim;
+}
+
+.buttonBackgroundDark {
+	background-color: $dark-ultra-dim;
+}
+
+.buttonText {
+	font-size: 17;
```

Comment:
We should make this font-size 14 based on a discussion Thomas had about Android styling. how is it looking on iOS? This is how it looks on Android.

Reply:
Nice catch! I think I missed that as well. Here's how it looks on iOS: 
<img width="300" src="https://user-images.githubusercontent.com/1845482/106290303-99100d00-624a-11eb-8506-3a3c92415c07.jpeg">
Will commit the sugesstion 👍